packages/react/src/file-upload/FileUploadExistingFile.tsx

```diff
@@ -43,7 +43,12 @@ export const FileUploadExistingFile = ({
 					</Box>
 					{href ? (
 						<Text breakWords paddingY={1.5}>
-							<TextLink href={href} rel="noopener noreferrer" target="_blank">
+							<TextLink
+								download
```

Comment:
Still needs to have `target="_blank"` otherwise it just tried to open it in the same tab